L1Trigger/L1TGlobal/interface/AXOL1TLCondition.h

```diff
@@ -83,6 +91,9 @@ namespace l1t {
     hls4mlEmulator::ModelLoader m_model_loader;
     std::shared_ptr<hls4mlEmulator::Model> m_model;
 
+    mutable std::array<float, NInputs> m_lastInputs_{};
```

Comment:
I am not sure exactly how this class's methods are currently called, but I am not sure if this usage is thread safe. Can you please retry your `cmsDriver.py` command with e.g. `--nThreads 8`?